R/partial.R

```diff
@@ -12,18 +12,26 @@
 #' similar as possible to the anonymous functions that you'd create by hand,
 #' if you weren't using `partial`.
 #'
-#' @param ...f a function. For the output source to read well, this should be a
-#'   named function.
+#' @param ...f a function.
 #' @param ... named arguments to `...f` that should be partially applied.
-#' @param .env the environment of the created function. Defaults to
-#'   [parent.frame()] and you should rarely need to modify this.
-#' @param .lazy If `TRUE` arguments evaluated lazily, if `FALSE`,
-#'   evaluated when `partial` is called.
+#'   [Unquoting and splicing][rlang::UQ()] semantics are supported, see
+#'   _Examples_.
 #' @param .first If `TRUE`, the partialized arguments are placed
 #'   to the front of the function signature. If `FALSE`, they are
 #'   moved to the back. Only useful to control position matching of
 #'   arguments when the partialized arguments are not named.
+#' @param .env,.lazy Obsolete, see _Details_.
```

Comment:
I think it would be more useful to extract the info out of details and re-include here.

Reply:
Makes sense, they go together.